src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -1023,112 +1044,114 @@ void func_80B8760C(EnKaizoku* this, PlayState* play) {
     }
 }
 
-void func_80B87900(EnKaizoku* this) {
-    EnKaizoku_ChangeAnim(this, EN_KAIZOKU_ANIM_7);
+void EnKaizoku_SetupBlock(EnKaizoku* this) {
+    EnKaizoku_ChangeAnim(this, KAIZOKU_ANIM_BLOCK);
 
     if (this->swordState != 0) {
         this->swordState = -1;
     }
-    this->unk_2D8 = 0;
+    this->dontUpdateSkel = false;
     this->picto.actor.speed = 0.0f;
-    this->unk_2B2 = Rand_S16Offset(10, 10);
+    this->combatTimer = Rand_S16Offset(10, 10);
     this->bodyCollider.base.acFlags |= AC_HARD;
     this->lookTimer = 12;
-    this->action = KAIZOKU_ACTION_12;
-    this->actionFunc = func_80B8798C;
+    this->action = KAIZOKU_ACTION_BLOCK;
+    this->actionFunc = EnKaizoku_Block;
 }
 
-void func_80B8798C(EnKaizoku* this, PlayState* play) {
+void EnKaizoku_Block(EnKaizoku* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
-    s16 temp_v0;
+    s16 angleToLink;
 
-    this->unk_2D8 = 1;
-    if (this->unk_2B2 == 0) {
+    this->dontUpdateSkel = true;
+    if (this->combatTimer == 0) {
         this->skelAnime.playSpeed = 1.0f;
     }
     if (this->lookTimer == 0) {
-        this->unk_2D8 = 0;
-        temp_v0 = this->picto.actor.yawTowardsPlayer - this->picto.actor.shape.rot.y;
-        if ((ABS_ALT(temp_v0) <= 0x4000) && (this->picto.actor.xzDistToPlayer < 40.0f) &&
+        this->dontUpdateSkel = false;
+        angleToLink = this->picto.actor.yawTowardsPlayer - this->picto.actor.shape.rot.y;
+        if ((ABS_ALT(angleToLink) <= 0x4000) && (this->picto.actor.xzDistToPlayer < 40.0f) &&
             (fabsf(this->picto.actor.playerHeightRel) < 50.0f)) {
             if (func_800BE184(play, &this->picto.actor, 100.0f, 10000, 0x4000, this->picto.actor.shape.rot.y)) {
                 if (player->meleeWeaponAnimation == PLAYER_MWA_JUMPSLASH_START) {
                     this->bodyCollider.base.acFlags &= ~AC_HARD;
-                    func_80B874D8(this, play);
-                } else if (!(play->gameplayFrames & 1)) {
-                    func_80B87900(this);
+                    EnKaizoku_SetupSpinDodge(this, play);
+                } else if (!(play->gameplayFrames % 2)) {
```

Comment:
```suggestion
                } else if ((play->gameplayFrames % 2) == 0) {
```
Much more of a preference when use modulos.